actions/invite_actions.jsx

```diff
@@ -74,7 +74,7 @@ export function sendGuestsInvites(teamId, channels, users, emails, message) {
         const members = getChannelMembersInChannels(state);
         for (const user of users) {
             if (!isGuest(user)) {
-                notSent.push({user, reason: localizeMessage('invite.members.user-is-not-guest', 'You can\'t invite a regular user as a guest.')});
+                notSent.push({user, reason: localizeMessage('invite.members.user-is-not-guest', 'This person is already a member.')});
```

Comment:
nit, I think referencing `user` instead of `person` sounds better, but 0/5:
```suggestion
                notSent.push({user, reason: localizeMessage('invite.members.user-is-not-guest', 'This user is already a member.')});
```

Reply:
Talked with @jespino offline and agreed on the texts coming from the designs. Resolving these comments